modules/search/instant-search/components/photon-image.jsx

```diff
@@ -0,0 +1,15 @@
+/** @jsx h */
+
+/**
+ * External dependencies
+ */
+import { h } from 'preact';
+import photon from 'photon';
+
+const PhotonImage = ( { src, maxWidth = 300, maxHeight = 300, alt, ...otherProps } ) => {
```

Comment:
@sirreal had worked on something a bit similar that included a few extra checks and handled `srcset`:
https://github.com/Automattic/jetpack/blob/354af3d0200913b05a4f650ee48e4e981b99dc29/extensions/blocks/tiled-gallery/utils/index.js#L18-L112

I wonder if it may be worth doing something similar here, or extract this so it can be used by both the block and Instant Search?

Reply:
Certainly seems like there's some generic utility there 👍

Reply:
@bluefuton, your thoughts on this?

Reply:
Yes - definitely agree it would be nice to break this out in the future and make it reusable 👍 I'll add it to the list.